From e11732dcc7d02a61bbd1208b1eb24afd8fc6d419 Mon Sep 17 00:00:00 2001
Message-ID: <e11732dcc7d02a61bbd1208b1eb24afd8fc6d419.1740173176.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1740173176.git.sharlatanus@gmail.com>
References: <cover.1740173176.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 17 Feb 2025 16:34:10 +0000
Subject: [PATCH 20/51] gnu: python-mpl-scatter-density: Update to 0.8.

* gnu/packages/python-science.scm (python-mpl-scatter-density): Update to 0.8.
[source] <uri>: Adjust it as PyPI archive name has been changed upstream.

Change-Id: Ic049ea153503cfa92c5d19b272afb00e64f7a28e
---
 gnu/packages/python-science.scm | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/python-science.scm b/gnu/packages/python-science.scm
index beff4353ce6..489ab461539 100644
--- a/gnu/packages/python-science.scm
+++ b/gnu/packages/python-science.scm
@@ -1049,13 +1049,13 @@ (define-public python-meshzoo
 (define-public python-mpl-scatter-density
   (package
     (name "python-mpl-scatter-density")
-    (version "0.7")
+    (version "0.8")
     (source
      (origin
        (method url-fetch)
-       (uri (pypi-uri "mpl-scatter-density" version))
+       (uri (pypi-uri "mpl_scatter_density" version))
        (sha256
-        (base32 "083n4xkwjmxvij9i1xhfnxz8vk39ay0baam4nf0khvcihw47bkna"))))
+        (base32 "0cynk1rk6k2xklgv69difphrz6id77x3xb58kbs4mc4q7z6bvfid"))))
     (build-system pyproject-build-system)
     (arguments
      (list
-- 
2.47.1

